### PR TITLE
Feature/product1

### PR DESCRIPTION
**Pull request by ThomasF85**
I created a product page, please check it and specifically look at the html file.

### CODE REVIEW

**Review by JudithRe — CHANGES_REQUESTED** [review]
Please add a markdown file.

**Comment by ThomasF85** [comment]
> Please add a markdown file.

Thanks for telling me. Please check again.

File: product1.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Product
+
+Nice description of my product
```
Comment (JudithRe) [inline product1.md:3]: Good job!

**Review by JudithRe — APPROVED** [review]
It looks good.

**Pull request merged**